src/integTest/groovy/com/readytalk/LifecycleIntegTest.groovy

```diff
@@ -10,7 +10,7 @@ class LifecycleIntegTest extends IntegrationSpec {
       apply plugin: 'com.readytalk.ci'
       apply plugin: 'java'
 
-      buildEnv.isCI = false
+      buildEnv.ci = false
```

Comment:
Speaking of 1.0, we should probably make it possible to use the ci publishing and lifecycle conventions without pulling in the nebula.info plugin, since the upstream plugin doesn't play nice with Java 6 projects, which are still pretty common and supported by Gradle.